recipes-devtools/rust/rust-source-1.30.0.inc

```diff
@@ -0,0 +1,22 @@
+# Specifics for Rust 1.30.0
+
+# SRC_URI += "https://static.rust-lang.org/dist/rustc-${PV}-src.tar.gz;name=rust"
+SRC_URI += " \
+        https://static.rust-lang.org/dist/rustc-${PV}-src.tar.gz;name=rust \
+        file://skip-crate-build-id-check.patch;patchdir=${WORKDIR}/rustc-${PV}-src \
```

Comment:
This patch file has not been committed, please update the PR to include it or drop it.